eZ/Publish/Core/IO/Exception/InvalidBinaryFileIdException.php

```diff
@@ -8,12 +8,22 @@
  */
 namespace eZ\Publish\Core\IO\Exception;
 
+use Exception;
 use eZ\Publish\Core\Base\Exceptions\InvalidArgumentValue;
 
 class InvalidBinaryFileIdException extends InvalidArgumentValue
```

Comment:
OK, but then my change should be partly reverted and we should instead get rid of the message part in order to not have strings as params.